lighthouse-core/audits/unused-css-rules.js

```diff
@@ -110,11 +111,16 @@ class UnusedCSSRules extends Audit {
       }
     }
 
-    return {
-      url: stylesheetInfo.header.sourceURL || 'inline',
-      label: `${percentUsed}% rules used`,
-      code: contentPreview.trim(),
-    };
+    let code;
+    let url = stylesheetInfo.header.sourceURL;
+    const label = `${percentUsed}% rules used`;
+
+    if (!url || url === pageUrl) {
```

Comment:
Oh whoops Brendan merged before I could :) it seems to be if it was in the original document or added later with script, which maybe a more sophisticated version later on could identify the URL of the script that added it too? I'll add the TODO in my upcoming rules -> bytes conversion PR for CSS.